helpers/addresses.py

```diff
@@ -498,6 +498,8 @@
     "across_bridge": {
         "hub_pool": "0x6Bb9910c5529Cb3b32c4f0e13E8bC38F903b2534",
     },
+    "maker": {
```

Comment:
forgot to mention that locally needed this little modification, was missing a curly bracket

```suggestion
    "maker": {
        "proxy_registry": "0x4678f0a6958e4D2Bc4F1BAF7Bc52E8F3564f3fE4",
    },
```

Reply:
pushed